WebRoot/js/ajax/dwt/widgets/DwtListView.js

```diff
@@ -508,6 +515,55 @@ function(defaultColumnSort, noResultsOk) {
 	this.removeAll();
 	this.createHeaderHtml(defaultColumnSort);
 	this._renderList(this._list, noResultsOk);
+	this._createScreenReaderTable();
+};
+
+DwtListView.prototype._createScreenReaderTable =
+function() {
+	if (!this.tableView) {
+		return;
+	}
+
+	var div = this.getHtmlElement();
+	var table = div.querySelector('table.ScreenReaderOnly');
+	if (table) {
+		table.remove();
+	}
+	var newTable = document.createElement('table');
+	newTable.className = 'ScreenReaderOnly';
+	var header = '<thead class="ScreenReaderOnly"><tr>';
+	for (var i = 0; i < this.column.length; i++) {
```

Comment:
`for (var i = 0, length = this.column.length; i < length; i++) {`

Reply:
Done.